src/chrome/content/rules/Open.ac.uk.xml

```diff
@@ -50,4 +50,11 @@
 
 	<rule from="^http:"
 			to="https:" />
+
+	<exclusion pattern="^http://(www\.)?open\.ac\.uk/courses(/.*)?$" />
```

Comment:
Isn't this equivalent to `<exclusion pattern="^http://(www\.)?open\.ac\.uk/courses" />` ?

Reply:
@cschanaj It would treat $ as a literal character. `(/|$)` will work though.